website/blog/2024-10-23-release-0.76-new-architecture.md

```diff
@@ -0,0 +1,224 @@
+
+The New Architecture is here. Starting from 0.76 the New Architecture is enabled by default in your projects.
+
+Thanks to the community's validation and testing, we now feel confident making this change. We verified that more than [880 libraries](https://reactnative.directory/?newArchitecture=true) are compatible with the New Architecture, including all the Expo Modules in the Expo SDK. We also worked with companies like [BlueSky](https://github.com/bluesky-social/social-app/releases/tag/1.92.0-na-rc.2) and [Kraken](https://blog.kraken.com/product/engineering/kraken-new-architecture) to validate the app migration, and they are already using it in production.
+
+This change will be completely transparent to you. Upgrade to version 0.76 by following the instructions in the [upgrade helper](https://react-native-community.github.io/upgrade-helper/?from=0.75.4&to=0.76.0), and the New Architecture will be enabled by default. We don’t expect any changes in behaviors in your app.
+
+If for any reason you can’t move to the New Architecture, you can still opt-out from it.
+
+- On Android, you can modify the `gradle.properties` file and turn the `newArchEnabled` flag to `false`.
+
+```diff
+-newArchEnabled=false
++newArchEnabled=false
+```
+
+- On iOS, you can opt-out from the New Architecture by installing CocoaPods dependencies with the command:
+
+```bash
+RCT_NEW_ARCH_ENABLED=0 bundle exec pod install
+```
+
+As part of the work to make the New Architecture the default, we also revamped most sections of the website to reflect these changes. If you need the documentation of the [Legacy Native Modules](/docs/next/legacy/native-modules-intro) and [Legacy Native Components](/docs/next/legacy/native-components-android), you can find it in the Legacy Architecture section. Otherwise, the rest of the website **assumes that you are looking for New Architecture documentation**.
+
+### React Native DevTools
+
+![React Native DevTools front-end](../static/blog/assets/0.76-react-native-devtools.jpg)
+
+We are shipping the first stable release of React Native DevTools, our new default debugging experience.
+We want the tooling you use to debug React across all platforms to be reliable, familiar, simple, and cohesive. React Native DevTools is an out-of-the-box experience that delivers on these principles — familiar browser-aligned developer tools that are deeply integrated with React Native.
+Key features:
+
+- **Familiar, web-aligned tooling** — A fully featured debugger based on Chrome DevTools, providing reliable breakpoints, watch values, step through debugging, stack inspection, and a rich JavaScript console. These core features now work reliably and across reloads.
+- **An improved and integrated version of React DevTools** — The built-in React Components Inspector and Profiler are powerful tools to inspect and understand your React app. Both tools work the same as they do on the web. Component highlighting is faster and more reliable than before.
```

Comment:
Can we keep the earlier wording for this? Helps maintain momentum in the sentence (at least, matches the writing style of the surrounding list items).

```suggestion
- **An improved and integrated version of React DevTools** — The built-in React Components Inspector and Profiler are powerful tools to inspect and understand your React app, and work just as they do on the web. Component highlighting is faster and more reliable than before.
```